strings.txt

```diff
@@ -17810,3 +17810,601 @@
+    ro = Mai multe
+    ar = المزيد
+    sk = Viac
+    es = Más
+    pl = Więcej
+    nl = Meer
+
+  [bookingcom_book_button]
+    en = Book
+    ru = Забронировать
+    fr = Réserver
+    da = Bog
+    id = Pesan
+    ko = 예약
+    sv = Boka
+    tr = Rezervasyon
+    uk = Забронювати
+    vi = Đặt trước
+    hu = Foglalás
+    de = Buchen
+    fi = Varaa
+    cs = Rezervace
+    it = Prenota
+    nb = Bestill
+    zh-Hans = 图书
+    th = จอง
+    ja = 書籍
+    ro = Rezervare
+    ar = حجز
+    sk = Rezervovať
+    es = Reservar
+    pl = Książka
+    nl = Boeken
+
+  [placepage_call_button]
+    en = Call
+    ru = Позвонить
+    fr = Appeler
+    da = Ring
+    id = Hubungi
+    ko = 전화
+    sv = Ring
+    tr = Çağrı
+    uk = Подзвонити
+    vi = Gọi
+    hu = Hívás
+    de = Anruf
+    fi = Soita
+    cs = Volat
+    it = Chiama
+    nb = Ring
+    zh-Hans = 呼叫
+    th = โทร.
+    ja = コール
+    ro = Apel
+    ar = اتصال
+    sk = Zavolať
+    es = Llamar
+    pl = Zadzwoń
+    nl = Bellen
+
+  [placepage_route_from_button]
+    en = Route from
+    ru = Отсюда
+    fr = Itinéraire de
+    da = Rute fra
+    id = Rute dari
+    ko = 출발지
+    sv = Rutt från
+    tr = Çıkış yeri:
+    uk = Звідси
+    vi = Tuyến từ
+    hu = Innen
+    de = Von
+    fi = Reitin alkupiste
+    cs = Cesta k
+    it = Percorso da
+    nb = Rute fra
+    zh-Hans = 发送自
+    th = เส้นทางจาก
+    ja = からのルート
+    ro = Ruta de la
+    ar = الطريق من
+    sk = Cesta z
+    es = Ruta desde
+    pl = Droga z
+    nl = Route van
+
+  [placepage_route_to_button]
+    en = Route to
+    ru = Сюда
+    fr = Itinéraire vers
+    da = Rute til
+    id = Rute ke
+    ko = 목적지
+    sv = Rutt till
+    tr = Varış yeri:
+    uk = Сюди
+    vi = Tuyến đến
+    hu = Ide
+    de = Nach
+    fi = Reitin loppupiste
+    cs = Cesta od
+    it = Percorso per
+    nb = Rute til
+    zh-Hans = 发送至
+    th = เส้นทางถึง
+    ja = へのルート
+    ro = Ruta la
+    ar = الطريق إلى
+    sk = Cesta do
+    es = Ruta hasta
+    pl = Droga do
+    nl = Route naar
+
+  [placepage_edit_bookmark_button]
+    en = Edit Bookmark
+    ru = Редактировать метку
+    fr = Éditer les signets
+    da = Rediger bogmærke
+    id = Edit Bookmark
+    ko = 즐겨찾기 편집
+    sv = Redigera bokmärke
+    tr = Yer İmini Düzenle
+    uk = Редагувати позначку
+    vi = Sửa Dấu Trang
+    hu = Könyvjelző szerkesztése
+    de = Lesezeichen bearbeiten
+    fi = Muokkaa kirjanmerkkiä
+    cs = Upravit záložku
+    it = Modifica segnalibro
+    nb = Rediger bokmerke
+    zh-Hans = 编辑书签
+    th = แก้ไข Bookmark
+    ja = ブックマークを編集
+    ro = Editare marcaj
+    ar = تحرير العلامة المرجعية
+    sk = Upraviť záložku
+    es = Editar marcador
+    pl = Edytuj zakładkę
+    nl = Bladwijzer bewerken
+
+  [placepage_bookmark_name_hint]
+    en = Bookmark Name
+    ru = Название метки
+    fr = Nom du signet
+    da = Bogmærke
+    id = Nama Bookmark
+    ko = 즐겨찾기 이름
+    sv = Namn bokmärke
+    tr = Yer İmi Adı
+    uk = Назва позначки
```

Comment:
+, и в одном старом переводе таки была “позначка”:)
> On Apr 26, 2016, at 4:56 PM, Alex Zolotarev <notifications@github.com> wrote:
> 
> In strings.txt <https://github.com/mapsme/omim/pull/3005#discussion_r61089415>:
> 
> > +    ar = تحرير العلامة المرجعية
> > +    sk = Upraviť záložku
> > +    es = Editar marcador
> > +    pl = Edytuj zakładkę
> > +    nl = Bladwijzer bewerken
> > +
> > +  [placepage_bookmark_name_hint]
> > +    en = Bookmark Name
> > +    ru = Название метки
> > +    fr = Nom du signet
> > +    da = Bogmærke
> > +    id = Nama Bookmark
> > +    ko = 즐겨찾기 이름
> > +    sv = Namn bokmärke
> > +    tr = Yer İmi Adı
> > +    uk = Назва позначки
> Назва мітки
> Везде в наших украинских переводах используется слово метка.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3005/files/454ca48abdc1a7a643a178fac869c249f100121f..9460d6ca1e36b2c17bee62d5d712319e41eb236f#r61089415>